framework/src/engine/rpc/rpc_server.ts

```diff
@@ -113,9 +117,11 @@ export class RPCServer {
 				(socket, message) => {
 					this._handleRequest(message)
 						.then(data => {
+							this._logger.info({ status: 'success' }, 'WSServer.start');
```

Comment:
This is not really about start, but it's for handling ws request. Same for http and IPC